.grunt-config/webpack.js

```diff
@@ -138,6 +138,7 @@ const entry = {
 	'styleguide': path.resolve( __dirname, '../modules/styleguide/assets/js/styleguide.js' ),
 	'styleguide-app-initiator': path.resolve( __dirname, '../modules/styleguide/assets/js/styleguide-app-initiator.js' ),
 	'e-home-screen': path.resolve( __dirname, '../modules/home/assets/js/app.js' ),
+	'atomic-opt-in': path.resolve( __dirname, '../modules/atomic-opt-in/assets/js/app.js'),
```

Comment:
I assume that you have discussed the naming convention with Omer. I am happy, as long we have Core team support on board.

Reply:
@mike-elementor Omer is currently working on naming conventions, and he is renaming some existing files. I think it will be a good thing if we ask him to check our module name. It won't be nice if it clashes with what he is working on, just to make sure that we are all aligned.